astropy/time/core.py

```diff
@@ -169,6 +169,65 @@ def _construct_from_dict(self, map):
 
         return out
 
+    def new_like(self, cols, length, metadata_conflicts='warn', name=None):
+        """
+        Return a new Time instance which is consistent with the
+        input ``cols`` and has ``length`` rows.
+
+        This is intended for creating an empty column object whose elements can
+        be set in-place for table operations like join or vstack.
```

Comment:
Since this is in the astropy.time module, it might not be clear what columns refers to. Maybe saying 'multiple time objects' would make more sense? Or maybe explain more about the Table mixin functionality here.